package.json

```diff
@@ -25,9 +25,13 @@
     "nunjucks": "^3.2.0",
     "path": "^0.12.7",
     "react": "16.8.6",
-    "react-dom": "16.8.6"
+    "react-dom": "16.8.6",
+    "react-dropdown": "^1.6.4",
+    "react-select": "^3.0.4"
   },
   "devDependencies": {
+    "@babel/core": "^7.5.5",
+    "@babel/preset-env": "^7.5.5",
```

Comment:
How does `@babel/core` with `@babel/preset-env` and `babel-core` with`babel-preset-env` work together?